x/auth/ante/ante.go

```diff
@@ -22,10 +23,11 @@ func NewAnteHandler(
 		TxTimeoutHeightDecorator{},
 		NewValidateMemoDecorator(ak),
 		NewConsumeGasForTxSizeDecorator(ak),
-		NewRejectFeeGranterDecorator(),
+		NewDeductGrantedFeeDecorator(ak, bankKeeper, feeGrantKeeper),
```

Comment:
This should be conditional. If feeGrantKeeper is nil use RejectFeeGrantDecorator.